src/equations/2d/linear_scalar_advection.jl

```diff
@@ -24,18 +24,27 @@ get_name(::LinearScalarAdvectionEquation2D) = "LinearScalarAdvectionEquation2D"
 varnames_cons(::LinearScalarAdvectionEquation2D) = SVector("scalar")
 varnames_prim(::LinearScalarAdvectionEquation2D) = SVector("scalar")
 
+function x_trans_periodic(x)
+  #Fixed domain_length and center.
+  domain_length = [2,2]
+  center = [0,0]
+  x = (x - center) .% (domain_length)
+  x += ((x .<= -0.5*domain_length) .- (x .>= 0.5*domain_length)) .* (domain_length)
```

Comment:
Maybe, for clarity, this can be written like so:
```suggestion
  @. x += ((x <= -0.5*domain_length) - (x >= 0.5*domain_length)) * domain_length
```
But I am not sure whether this will change semantics. Also, wouldn't it suffice to use `<` and `>` instead of `<=` and `>=`?

Reply:
Yes, < and > will suffice and I´ll use them instead.
@. looks like a great way to make expressions more compact, but I´m planning to use SVectors and I´m not sure if @. works with them.

Reply:
Yes, `@.` works with `SVector`s.